src/actions/wincher/wincher-account-action.php

```diff
@@ -63,4 +64,26 @@ public function check_limit() {
 			];
 		}
 	}
+
+	/**
+	 * Gets the upgrade campaign.
+	 *
+	 * @return object The response object.
+	 */
+	public function get_upgrade_campaign() {
+		try {
+			$result = $this->client->get( self::UPGRADE_CAMPAIGN_URL );
+			$discount = $result['value'];
```

Comment:
We don't support AMOUNT discounts here. Maybe that's fine but we should then ignore amount discounts returned from the API just in case. 

However, you need to return and use the number of months for the discount since that might change.

Reply:
Support of months added.

- Months returned by the API:
![image](https://user-images.githubusercontent.com/129404933/231828885-54279596-4bbc-4dcb-b62d-9b3947a0c2e5.png)

- No campaign:
![image](https://user-images.githubusercontent.com/129404933/231829996-a1091c7f-7f21-4707-8ddd-9a076aa6f67b.png)

- Amount-type campaign:
![image](https://user-images.githubusercontent.com/129404933/231830568-88619070-c50f-4469-ae88-534e43757cf5.png)

- Same for `months = null` and `months = 0`